src/config.ts

```diff
@@ -174,6 +174,16 @@ export const CHUNK_POST_MIN_SUCCESS_COUNT = +env.varOrDefault(
   '3',
 );
 
+// Contribute to a healthy Arweave Network by posting chunks to peers
+// default value of 0 means this behaviour is disabled.
+export const ARWEAVE_PEER_CHUNK_POST_MIN_SUCCESS_COUNT_STRING =
```

Comment:
The `_STRING` suffix on both environment variables should be removed. We only use that in cases where we are preserving the original string value rather converting it.

Reply:
fixed